pkg/kademlia/kademlia.go

```diff
@@ -275,16 +283,23 @@ func (k *Kademlia) FetchInfo(ctx context.Context, node pb.Node) (_ *pb.InfoRespo
 	return info, nil
 }
 
-// FindNode looks up the provided NodeID first in the local Node, and if it is not found
-// begins searching the network for the NodeID. Returns and error if node was not found
+// FindNode wraps a call to FindNodeWithAntechamber with 0 antechamber nodes
```

Comment:
why are we just wrapping FindNodeWithAntechamber?

Reply:
Every instance of FindNode, we never want to be given any antechamber nodes. I could have just added the `antechamberLimit` argument to FindNode, but it seemed weird to me to have to specify an argument that we know we don't want except for one place: node discovery. With FindNode as a wrapper, this allows us to leave all the existing FindNode calls alone